EIPS/eip-FOCIL.md

```diff
@@ -26,34 +27,32 @@ FOCIL is a simple committee-based design improving upon previous IL mechanisms o
 
 ## Specification
 
-### Consensus layer
+### Execution Layer
+
+TBD
 
-#### High-level overview
+### Consensus Layer
 
 #### Timeline
 
-A set of validators is selected from the beacon committee to become IL committee members for `slot N`.
+A set of validators is selected from the beacon committee to become IL committee for `slot N`. Their local inclusion lists constraint the block proposer for `slot N+1`.
 
-- **`Slot N`, `t=6`**: The IL committee of `slot N+1` releases local ILs, knowing the contents of `block N`. Local ILs are sets of full transactions pending in the public mempool.
-- **`Slot N`, `t=9`**: Local IL freeze deadline, after which all attesters of `slot N+1` lock their view of the observed local ILs. The `slot N+1` proposer broadcasts a signed IL aggregate constructed by deduplicating and taking the union of transactions from collected local ILs. Every IL aggregate (`from_address`, `gas_limit`) is associated with a `bitlist` field indicating which IL committee member included a given transaction.
-- **`Slot N+1`, `t=0`**: The block producer of `slot N+1` releases its `block B`, which contains both the payload and the IL aggregate.
-- **`Slot N+1`, `t=4`**: The attesters of `slot N+1` vote on `block B`. `Block B` is considered valid if:
-  - The IL aggregate included in `block B` corresponds to the attesters' local view of available local ILs and is the same IL aggregate broadcast during `slot N`
-  - The IL aggregate entries are satisfied by payload transactions
+- **`Slot N`, `t=0 to 9s`**: After processing the block for `slot N` and confirming it as the head, each IL committee member of `Slot N` constructs a local inclusion list based on the head and their view of the public mempool, then broadcasts it over the P2P network. If no block arrives by `t=8s`, each IL committee member should run `get_head` to retrieve their canonical head and use it to build their local inclusion list.
```

Comment:
change from `t=0 to 9s` to `t=0 to 8s`

Reply:
9s is correct as IL committee can broadcast their local IL between 8s and 9s, no?

Reply:
I know the idea but isn't the IL deadline still 9s? The description between 0-9s follows IL committee members section in the [post](https://ethresear.ch/t/focil-cl-el-workflow/20526).

Reply:
Yeah yeah I know it's different than the post, but after discussions we think it's better to have a buffer so local Ils have time to propagate before the freeze deadline (1 sec should be enough)

Reply:
Ah, so there was some change that I haven't heard of. Applied it, thanks.